contracts/mocks/Assessment/ASMockIncidents.sol

```diff
@@ -44,7 +44,7 @@ contract ASMockIncidents is MasterAwareV2 {
       priceBefore
     );
 
-    uint expectedPayoutInNXM = activeCoverAmountInNXM * config.incidentExpectedPayoutRatio /
+    uint expectedPayoutInNXM = activeCoverAmountInNXM * config.expectedPayoutRatio /
```

Comment:
i removed the ability to read activeCoverAmountInNXM for a particular product. we'd need a parameter for this